### ISSUE TITLE
Asset Click New Media Library Folder No Response

### ISSUE THREAD

**Issue by pgy866** [OrchardCMS/OrchardCore#1039]
Asset Click New Media Library Folder No Response
https://github.com/OrchardCMS/OrchardCore/issues/1009

![image](https://user-images.githubusercontent.com/28694676/30954516-033c25d2-a463-11e7-998c-2033352a1511.png)

**Comment by agriffard** [comment]
It works for me, I can open the popup :
![image](https://user-images.githubusercontent.com/703248/30965123-801fb050-a454-11e7-9125-f9d7027536b0.png)
Which browser do you use ? Do you see a javascript error if you open your console (F12) and retry?

But then, I have an error if I want to create an 'img' folder. See the web console errors :
![image](https://user-images.githubusercontent.com/703248/30965250-0167e650-a455-11e7-85dc-b360496029e2.png)

**Comment by agriffard** [comment]
@sebastienros :
The creation of a folder doesn't work since this change : 2d3a0f4bb0ea13cf4c340cd336e4f67e31f56bf6 

This line :
`-                Directory.CreateDirectory(GetPhysicalPath(subpath));`

has been replaced by this one:
```
 +                // Use CreateSubdirectory to ensure the directory doesn't go over its boundaries
 +                new DirectoryInfo(_localPath).CreateSubdirectory(subpath);
```

but the subpath is transmitted like this '`/foldername`' so it is not the expected format to create a sub directory.

In fact the GetPhysicalPath and NormalizePath methods are not correct because they keep the folder with '/' characters instead of replacing them by '\\'.

**Comment by pgy866** [comment]
<Google Chrome  61.0.3163.100>
I do not show the wrong situation
![image](https://user-images.githubusercontent.com/28694676/31011757-07dd6062-a542-11e7-85f6-4e63f34b590b.png)

**Comment by pgy866** [comment]
"Google Chrome 61.0.3163.100" + gray, can not create a new folder
![image](https://user-images.githubusercontent.com/28694676/31012016-e784bd64-a542-11e7-9b43-8f3d3bc80db3.png)

**Comment by pgy866** [comment]
@agriffard 
Thank you very much for your answer，
My English is very bad, use Google translation, hope you can understand。

**Comment by pgy866** [comment]
MediaField and Image Widget can not be used, do not know whether it is relevant。
![image](https://user-images.githubusercontent.com/28694676/31012544-bf70041c-a544-11e7-8618-c88275dbfdf4.png)
![image](https://user-images.githubusercontent.com/28694676/31012580-e0a1163a-a544-11e7-80b5-2a5180e982e6.png)

**Comment by agriffard** [comment]
@pgy866 : you have an error to retrieve this file : https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js so jQuery is not loaded.

Can you open this url in your browser?

**Comment by pgy866** [comment]
Do not show the error,
Really do not open!
May show that,
Is because a plug-in proxy network friends,
But can not create a folder that is present
![6](https://user-images.githubusercontent.com/28694676/31016599-f5e551c4-a557-11e7-9c56-41dd7c1cfdd3.gif)

**Comment by agriffard** [comment]
OK, now you get jQuery loaded but the remaining error is the one I described:
The creation of a Media folder needs to be fixed.

**Comment by pgy866** [comment]
拥抱！! Thank you very much！agriffard同学！

**Comment by pgy866** [comment]
Can  consider adding "asset" rights management? And even can precise control one of the file access, modify, delete and other permissions settings？
